WordPress/Classes/ViewRelated/Site Creation/Design Selection/SiteDesignContentCollectionViewController.swift

```diff
@@ -289,7 +289,7 @@ extension SiteDesignContentCollectionViewController: UITableViewDataSource {
         cell.clipsToBounds = false
         cell.collectionView.allowsSelection = !isLoading
         cell.categoryTitleFont = WPStyleGuide.serifFontForTextStyle(.title2, fontWeight: .semibold)
-        cell.categoryCaption = sections[safe: indexPath.row]?.caption
+        cell.categoryCaption = cell.section?.caption
```

Comment:
Thanks for addressing this @Gio2018! I could've been clearer in my original comment.

Since we know that a `CategorySection` will have a [caption property](https://github.com/wordpress-mobile/WordPress-iOS/pull/18710/files#diff-fc495c370c2f62da0a661a8555dc581db91242d1ff11daaa0a73c2dd55fedaf3R20), could we simplify this even further by:
- Removing the ability to explicitly set a `cell.categoryCaption` (unless we think we need it)
- Relying on the `CategorySection` [setter](https://github.com/wordpress-mobile/WordPress-iOS/pull/18710/files#diff-fc495c370c2f62da0a661a8555dc581db91242d1ff11daaa0a73c2dd55fedaf3R50) of the cell to set the caption

This isn't a blocker to me and I could certainly be overlooking something, but wanted to bring it up. 👍

Reply:
Whooppsss 🤦 sorry I didn't understand the comment. For sure, you're 100% right, and thanks for catching this!